assets/tailwind.config.js

```diff
@@ -13,6 +23,15 @@ module.exports = {
       invert: {
         50: ".50",
       },
+      spacing: {
+        50: "12.5rem",
+      },
+      inset: {
+        ...generateFractionPercentages(20),
+      },
+      height: {
+        ...generateFractionPercentages(10),
+      },
```

Comment:
Why do we need to generate this? Isn't the [arbitrary values syntax](https://tailwindcss.com/docs/height#arbitrary-values) enough in that case?

Reply:
You are right. Arbitrary values are good enough. I didn't notice that syntax when I was working on it..